ProjFS.Mac/PrjFSKextLogDaemon/PrjFSKextLogDaemon.cpp

```diff
@@ -155,6 +165,123 @@ static os_log_type_t KextLogLevelAsOSLogType(KextLog_Level level)
     }
 }
 
+static void LogPanics()
+{
+    DIR *dr = opendir(PanicLogDirectory);
```

Comment:
We've been using C++-style pointer asterisk placement so far:
```
    DIR* dr = opendir(PanicLogDirectory);
```

Any reason to deviate from this here?

Reply:
There are several pointer declarations like this, could you update all of them?